fe/src/pages/Home/Home.tsx

```diff
@@ -128,56 +129,56 @@ const PRODUCTS_CARD_ITEMS: Array<ProductCardItem> = [
 ];
 
 function Home(): JSX.Element {
-  const [viewType, setViewType] = useState(ViewType.Grid);
+  const [view, setView] = useState(ViewType.Grid);
+  const homeContextProvider = useMemo(() => ({ view, setView }), [view, setView]);
+
   const isViewSwitchable: boolean = useBreakpointValue({ base: false, lg: true }) || false;
 
-  if (!isViewSwitchable && (viewType !== ViewType.Grid)) {
-    setViewType(ViewType.Grid);
+  if (!isViewSwitchable && (view !== ViewType.Grid)) {
+    setView(ViewType.Grid);
   }
 
   return (
-    <Box
-      px={2}
-      py={16}
-    >
-      <Box display="inline-block" p={2} w="100%">
-        <ViewToggler
-          view={viewType}
-          setView={setViewType}
-          viewSwitchable={isViewSwitchable}
-        />
-      </Box>
+    <HomeContext.Provider value={homeContextProvider}>
```

Comment:
@Orzelius Would it make sense to do something like this in [App.tsx](https://github.com/TPT-Loane/TPT-Loane/blob/main/fe/src/App.tsx#L15), instead of creating an extra parent component?
```
        <Default>
          <Switch>
            <HomeContext.Provider value={homeContextProvider}>
              <Route exact path="/" component={Home} />
            </HomeContext.Provider>
            ...
          </Switch>
        </Default>
```

Reply:
Yeah I'm not sure if it will work like that. Also the context would have to be exported outside of home then.

Reply:
Sorry for not updating this thread, still getting used to Github. You can see below my updated comment -> https://github.com/TPT-Loane/TPT-Loane/pull/79#issuecomment-838468791